### ISSUE TITLE
A11y statement dependant on markup for compliance string

### ISSUE THREAD

**Issue by marcusosterberg** [Webperf-se/webperf_core#1038]
### URL

https://exempel-saknas.se

### Webperf_core version

2025.3.7

### What environment are you running?

- [ ] Using GitHub Actions
- [ ] Using Docker Image from Dock Hub
- [x] Local machine
- [ ] Other, please describe your environment in How to reproduce.

### Has anyone else reported it already?

- [x] No, there are no other issues related to this in [Issues](https://github.com/Webperf-se/webperf_core/issues).

### Expected Behavior

If the a11y statement's text is "delvis förenlig" it should be noted by the test to be partially compliant.

### Actual Behavior.

If the text has markup like:
`Den här webbplatsen är <span>delvis</span> förenlig`
then the test thinks like this:
`Saknar helt formulering om följsamhet till lagkraven ( 1.00 betyg )`

### How to reproduce

```shell
Try a local a11y statement with the following string in the HTML:
Den här webbplatsen är <span>delvis</span> förenlig
```

### Log output

```shell

```
